setup.py

```diff
@@ -1,50 +0,0 @@
-from setuptools import find_packages, setup
-
-INSTALL_REQUIRES = [
-    'flask',
-    'gunicorn',
-    'datacube',
-    'altair',
-    'altair_saver',
-    'python-dateutil',
-    'sentry_sdk[flask]',
-    'blinker',
-    'prometheus-flask-exporter',
-    'pywps',
-    'pyarrow'
-]
-
-DESCRIPTION = ("datacube-wps is an implementation of the Web Processing Service standard "
-               "from the Open Geospatial Consortium.")
-
-KEYWORDS = 'ODC WPS OGC processing'
-
-config = {
-    'description': DESCRIPTION,
-    'keywords': KEYWORDS,
-    'author': 'Digital Earth Australia',
-    'license': 'Apache License 2.0',
-    'platforms': 'all',
-    'url': 'http://www.ga.gov.au/dea',
-    'download_url': 'https://github.com/opendatacube/datacube-wps',
-    'author_email': 'earth.observation@ga.gov.au',
-    'maintainer': 'Digital Earth Australia',
-    'maintainer_email': 'earth.observation@ga.gov.au',
-    'classifiers': [
-        'Development Status :: 4 - Beta',
-        'Environment :: Web Environment',
-        'Intended Audience :: Developers',
-        'Intended Audience :: Science/Research',
-        'License :: OSI Approved :: Apache Software License',
-        'Operating System :: OS Independent',
-        'Programming Language :: Python',
-        'Topic :: Scientific/Engineering :: GIS'
-    ],
-    'install_requires': INSTALL_REQUIRES,
-    'setup_requires': ['setuptools_scm'],
```

Comment:
We've lost setuptools_scm in the conversion.

Reply:
@omad seems the PyPi workflow only runs on merge, so this also broke that. You did something that removed a bunch of code in https://github.com/opendatacube/datacube-core/pull/1713, can the same be done for this repository?

Reply:
Yep to both, I'll have a look tomorrow.